ddtrace/debugging/_probe/model.py

```diff
@@ -241,7 +243,7 @@ class SpanProbeMixin(object):
 
 @attr.s
 class SpanFunctionProbe(Probe, FunctionLocationMixin, SpanProbeMixin, ProbeConditionMixin):
-    pass
+    __context_mutator__ = True
```

Comment:
SpanDecorationProbe are also context-mutating...
though, I need to execute SpanProbe first VS SpanDecorationProbe

Reply:
Good point. Here I'm regarding the context as the object that bears the correlation IDs essentially. Mutating the tags on a span is mutating the span itself, and I'm not regarding the span as a context, even though I think the tracer actually does). Perhaps the more correct terminology would be "creation" rather than "mutation", as essentially a span probe _creates_ a new context, rather than mutate (it mutates the context _hierarchy_).